code/socialDistribution/models/post.py

```diff
@@ -319,3 +320,64 @@ class InboxPost(Post):
     def author_as_json(self):
         """ Gets the author of the post in JSON format. """
         return self._author_json
+    
+    def fetch_update(self):
+        """ Fetches update about the post for an edit or delete """
+        # make api request
+        try:
+            actor_url = self.author.strip('/')
+            object_url = self.public_id.split('/')[-1]
+            endpoint = actor_url + '/posts/' + object_url
```

Comment:
Minor comment, can we make sure we only do this for public posts?

Private posts are likely to give us a 404 anyways, which means they will delete before we ever get to see them.

Just add an early exit if a post is not public. Thanks